configs/machines/levante.yaml

```diff
@@ -308,29 +331,14 @@ computer:
                 # kh 17.02.22 orig mistral, not yet tested on levante (but should work)
                 PERL5LIB: /usr/lib64/perl5
 
-                LD_LIBRARY_PATH[(0)]: $HDF5ROOT/lib:$NETCDFROOT/lib:$NETCDFFROOT/lib:$LD_LIBRARY_PATH
-
-                # avoid GLIBCXX_3.4.15 not found error
-                # MA: This solves nothing for AWICM3
-                #LD_LIBRARY_PATH[(1)]: /sw/spack-levante/gcc-11.2.0-bcn7mb/lib64:$LD_LIBRARY_PATH
-
-                # kh 01.03.22 current philosophy on levante regarding the modules
-                # concept is to avoid LD_LIBRARY_PATH as far as possible to make
-                # everything clearer
-
-                # kh 17.02.22 todo (not yet required for component fesom-2.1-recom-par_tracers)
-                #       LD_LIBRARY_PATH[(1)]: ...
-
-                # The following line is also moved to echam.yaml, only active for mistral and paleodyn echam6:
-                # - "LD_LIBRARY_PATH=/sw/rhel6-x64/netcdf/parallel_netcdf-1.6.1-impi-intel14/lib/:$LD_LIBRARY_PATH"
+                LD_LIBRARY_PATH[(0)]: $HDF5ROOT/lib:$NETCDFROOT/lib:$NETCDFFROOT/lib:$AEC_ROOT/lib:$LD_LIBRARY_PATH
```

Comment:
I don't think this would break anything for AWI, but let's check. Otherwise, we will need to move the changes required by icon into ICON itself. To be discussed with @seb-wahl